src/frameworks/flask/index.ts

```diff
@@ -13,7 +15,18 @@ export const type = FrameworkType.Framework;
 export async function discover(cwd: string) {
   if (!hasPipDependency("Flask", { cwd })) return;
   const results = await getDiscoveryResults(cwd).catch(() => undefined);
-  if (!results) return;
+  if (!results) {
+    logger.debug(
+      "It seems that the codebase is a Flask project. Here're some tips that will help us to sucessfully initialize a Flask project:"
+    );
+    logger.debug("\t1. Make sure that your app is in a main.py file in the root of your project.");
+    logger.debug(
+      '\t2. You have created and activated and virtual environment "python -m venv venv && . venv/bin/activate"'
+    );
+    logger.debug('\t3. You have run "pip install -t requirements.txt"');
```

Comment:
I defer to @alexastrum for the copy here

Reply:
I like the step by step instructions.

I rewrote some instruction:
1. Removed passive voice.
    We generally want to avoid this style of writing, and instead stick with a **Firebase voice** style - approachable, enthusiastic, and friendly.
    In future, it's probably best to run copy text by one of our awesome tech writers, @egilmorez, since I'm not really a wordsmith myself.
    
2. Explicitly pointed out that Python source file should reside in the hosting root subfolder (since root might also be interpreted as top level, eg the folder with `firebase.json` config).
    I'd like us to opinionated in keeping web frameworks code within the boundaries of `hosting`, since Firebase also supports Cloud Functions, that might have their own `*.py` files. Of course some users might choose `.` as hosting root, but I really want to discourage this.

```js
logger.debug(
      "Looks like you might be using Flask in your project. Here're some tips that will help our tools work with your Flask project:"
    );
    logger.debug('\t1. You have your app entry point in a "main.py" file in the hosting root folder.');
    logger.debug(
      '\t2. You have created and activated and virtual environment "python -m venv venv && . venv/bin/activate"'
    );
    logger.debug('\t3. You have run "pip install -t requirements.txt" and are able to start a standalone Flask server');
```